### ISSUE TITLE
Writing challenges we met as a group for Milestone 2 Report

### ISSUE THREAD

**Issue by 38programmer61** [bounswe/2021SpringGroup6#101]
Carrying out a project and preparing a milestone report is a very serious and beneficial actions, so being able to list the lessons learned and difficulties encountered is very useful for others especially when they also get into a similar task due to the fact that they will, probably, experience similar problems. Consequently, express the challenges met during the execution of the [Implementation Assignment](https://moodle.boun.edu.tr/mod/assign/view.php?id=409101) and the preparation of its report. Also don't forget to make proper classification and use neat language.

**Comment by 38programmer61** [comment]
Main points about the challenges encountered has been grouped and added as different paragraphs to the [Milestone 2 Report File](https://docs.google.com/document/d/18h_tSLL-ozRxbWTddxjbscwqDCv9LiqVZV7egKG3mhk/edit#) under Challenges We Faced As A Group title and it can be seen in the github when the last version of the document is uploaded to the GihHub repository. Also I need to mention that when I have looked at the document, I noticed that there're already some content in that part, so I have added my sentences on below the available ones. Approximately half of the content belongs to me and my part starts with "In addition to these points" and goes until the end.
